content/ADR-206-backpack-v2-back-end-dependencies.md

```diff
@@ -0,0 +1,278 @@
+The root cause of the issue comes from trying to treat different wearable types as if they were the same, when they are actually quite different. The forcing of this into similar-looking elements adds a lot of complexity to implementation and maintenance of the UI also.
+
+#### Option 2 - Multiple endpoints - Front end code unifies
+Leverage the existing endpoints created for passport, enhancing them with filtering and sorting capabilities. Change would be backwards compatible, it only adds specification for filtering and sorting criteria, but still behaves the same when nothing is specified.
+This solution doesn’t serve exactly what the new UI design suggests, but the endpoints can still be used to provide the same approach (unified view) by implementing the unification in the client side.
+The main advantages of this approach are:
+- Clean API design.
+- Very fast API dev time, as the endpoints already exist and work only involves adding support for filtering / sorting.
+
+The disadvantage is that the UI needs to either implement the unification by consuming the existing endpoints + adding a layer of merging the different sources into a single collection.
+
+#### Option 3 - Multiple endpoints - UI changes to show items separately (recommended)
+This option is similar to Option 2 in regards to the API endpoints: enhancing existing endpoints with filtering / sorting capabilities.
+The difference would be that the UI changes to show the 3 types of wearables separately. So this makes the nature of the different types of wearables (which have different behavior) become visible to the user, rather than trying to hide it.
+The UI should be iterated and adapted so it doesn’t try to force the unified view of entities of different kinds. It will be easier for users to understand what is happening. It will be easier for UI developers to develop and maintain in the future.
+For e.g. users may wonder: why can’t I see transactions for this wearable (being a third party one)? Why are base wearables at the beginning / end when sorting by date? Or other things that make no sense in one type but do in others.
+
+#### Option 4 - Unified intermediate service (CHOSEN OPTION)
+This option aims to have an intermediary service, out of the Decentraland protocol, and enable use cases for our reference client.
+In this approach we would be using the endpoints resulting from Option 3 in the service for wearables and doing the merge of the three while paginating the results to the client. In this case the solution in the frontend side would be similar to Option 1, having a single endpoint for paginating all kinds of wearables.
+It remains to be analyzed where this intermediate service would be hosted, whether inside or outside the Catalysts.
+
```

Comment:
What if we add a `Conclusion` section here saying why the option 4 was chosen?

Reply:
Done.